src/main/resources/swagger.api/schemas/tenant.yaml

```diff
@@ -5,8 +5,8 @@ Tenant:
       type: string
     code:
       type: string
-      minLength: 3
-      maxLength: 3
+      minLength: 2
+      maxLength: 5
```

Comment:
Do we need to adjust karate tests?

Reply:
If existing tests passing, introducing new for this change is not necessary

Reply:
karate tests need to be updated